service/src/main/java/org/ehrbase/aql/sql/QueryProcessor.java

```diff
@@ -35,6 +35,7 @@
 import org.ehrbase.dao.access.interfaces.I_DomainAccess;
 import org.ehrbase.service.IntrospectService;
 import org.jooq.*;
+import org.jooq.Record;
```

Comment:
Minor but we should either go with wildcard imports or either remove them but I think that we have to keep consistency in the code.

From personal point of view, I prefer the second option. In addition, it is compliant with Google Java style guide.

Reply:
I did this change as it was causing a compile error with the IDE (but not with Maven...). Basically, it was conflicting with a java.lang.Record. Not quite sure what is the issue TBH

Reply:
You are right (One more reasons to avoid wildcard imports 😛 )